hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -0,0 +1,510 @@
+"""
+Script to generate a dataset and perform hand-eye calibration using a Universal Robot UR5e robot.
+The script communicates with the robot through TCP/IP.
+
+The entire sample consist of two additional files:
+    - universal_robots_hand_eye_script.urp: Robot program script that moves between different poses.
+    - robot_communication_file.xml: communication set-up file.
+
+Running the sample requires that you have universal_robots_hand_eye_script.urp on your UR5e robot,
+and robot_communication_file.xml in the same repo as this sample. Each robot pose
+must be modified to your scene. This is done in universal_robots_hand_eye_script.urp on the robot.
+
+Further explanation of this sample is found in our knowledge base:
+https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/103481384/
+"""
+import argparse
+from pathlib import Path
+import time
+import datetime
+
+import cv2
+import numpy as np
+from scipy.spatial.transform import Rotation as R
```

Comment:
I think using the shorthand "R" here is unnecessary. It just adds another possible layer of confusion.

Reply:
done